optimism/src/mips/witness.rs

```diff
@@ -655,8 +655,7 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
             self.keccak_env = Some(keccak_env);
 
             // Reset environment
-            self.preimage_bytes_read = None;
-            self.preimage = None;
+            self.preimage_bytes_read = Some(0);
```

Comment:
Why is it an option and not `0`?

Reply:
Opening https://github.com/o1-labs/proof-systems/issues/1702

Reply:
Yeah good point... Actually it makes more sense that preimage_bytes_read gets initialized to Some(0) when reading the preimage key, and then here this becomes a None... Otherwise there's no difference with just an int.